force-app/main/default/classes/DataBuilder.cls

```diff
@@ -222,5 +225,44 @@ public with sharing class DataBuilder {
         return telemetryList;
     }
 
+    private Map<String, Object> buildPersonData(Item item) {
+        if (item.isUncaught) {
+            // The uncaught error handlers won't have anything meaningful at
+            // UserInfo. Use the provided personId instead.
+            return buildPersonDataFromId(item.personId);
+        }
+
+        return buildCurrentPersonData();
+    }
+
+    private Map<String, Object> buildCurrentPersonData() {
+        Map<String, Object> data = new Map<String, Object>();
+
+        data.put('id', UserInfo.getUserId());
+        data.put('username', UserInfo.getUserName());
+        data.put('email', UserInfo.getUserEmail());
```

Comment:
Where is `UserInfo` set or defined?

Reply:
It's a system provided service. https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_methods_system_userinfo.htm

Reply:
I see! Thanks!